### PR TITLE
Jetpack Social: Update readme

### PR DESCRIPTION
**Pull request by danielpost**
This PR updates the readme.txt file of Jetpack Social with the latest changes and formatting fixes.

#### Changes proposed in this Pull Request:

* Removed the donation link
* Added Automattic as the first contributor
* Removed the author tag, that way the author will just show as Automattic
* Slightly changed the installation instructions to match Jetpack Backup’s
* Fixed the formatting for the FAQ questions
* Updated the screenshot captions (pbNhbs-2JK-p2#comment-8438)
* Changed the PHP and version requirements to match Jetpack’s

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

N/A

### CODE REVIEW

**Review by pablinos — COMMENTED** [review]
Looks good. Just had a couple of thoughts reading through it.

File: projects/plugins/social/readme.txt [hunk 1]
```diff
@@ -14,54 +13,82 @@ Write once, publish everywhere. Reach your target audience by sharing your conte
...
 
+The first option is to install Jetpack Social from within your WP Admin.
+
 1. To begin, click on the Plugins link in the left hand sidebar, then click Add New.
-2. Search for Jetpack Social. The latest version will be in the search results. Click the Install Now button:
-3. Next, click the Activate button. After activating, you will be prompted to set up Jetpack Social.
+2. Search for Jetpack Social. The latest version will be in the search results. Click the Install Now button.
+3. Next, click the Activate button. After activating, you can navigate to "Jetpack → Social" in your admin area.
+
+### Manual Alternatives
```
[the remark this answers is not in the document]
Comment (danielpost) [reply projects/plugins/social/readme.txt:38]: Nope, none do. In general it seems like we could streamline these instructions a bit, they're all slightly different for each plugin.

File: projects/plugins/social/readme.txt [hunk 2]
```diff
@@ -14,54 +13,82 @@ Write once, publish everywhere. Reach your target audience by sharing your conte
...
 
 == Frequently Asked Questions ==
 
-### How do I connect to social networks?
+= How do I connect to social networks? =
+
+From your site’s WP Admin:
+
+1. Navigate to Jetpack → Social.
+2. Click Connect next to the social network you want to connect to.
```
[the remark this answers is not in the document]
Comment (danielpost) [reply projects/plugins/social/readme.txt]: Good catch! Fixed.

File: projects/plugins/social/readme.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (pablinos) [reply projects/plugins/social/readme.txt:38]: Yeah, I think you're right. The process will be largely the same for each of the plugins.

**Review by pablinos — APPROVED** [review]
This looks good to me!

**Pull request merged**